openstack/blockstorage/extensions/volumeactions/results.go

```diff
@@ -157,6 +158,14 @@ type VolumeImage struct {
 	// Current status of the volume.
 	Status string `json:"status"`
 
+	// Visibility defines who can see/use the image.
+	// supported since 3.1 microversion
+	Visibility images.ImageVisibility `json:"visibility"`
```

Comment:
Let's change this to just `string`. All results should be basic Golang types. This helps prevent any issue if the returned value is something custom or something we haven't accounted for.

(there might be some non-basic types hanging around other `results.go` files - these were either added prior to this decision being made or have slipped by)

Reply:
done